tests/dummy/app/templates/application.hbs

```diff
@@ -1,8 +1,17 @@
 <div class="padding-sm">
   <HyperTableV2 @handler={{this.handler}}>
-    <:search>
+    {{!-- <:search>
       <OSS::InputContainer style="width: 200px" @value={{this.searchQuery}}
                            @placeholder="Custom Search Block" @onChange={{this.onCustomSearchInput}} />
-    </:search>
+    </:search> --}}
```

Comment:
As explained vocally, there is a default search bar enabled when no <:search> named block is passed ; since this is the test application it's fine to leave this comment for now)